discrete_math/euclidean_algorithm/README.md

```diff
@@ -0,0 +1,149 @@
+# Dokumentacja projektu: Rozszerzony Algorytm Euklidesa

```

Comment:
You still need to add description how you find coefficients, the recursive formula is not gcd(gcd(gcd(...))) but gcd(gcd(a_1, ..., a_n-1), a_n) . If you do not provide it  then -1point is taken from you.